Traceback (most recent call last):
  File "/home/pakwan/cbTracker/pkg/tools.py", line 39, in read_json
    with open(path, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'resources/data/validation.json'
